lithops/serverless/backends/code_engine/code_engine.py

```diff
@@ -239,6 +241,31 @@ def list_runtimes(self, docker_image_name='all'):
 
         return runtimes
 
+    def clear(self):
+        """
+        Clean all completed jobruns
+        """
+        logger.debug('Deleting all completed jobruns')
+        jobruns = []
+        try:
+            jobruns = self.capi.list_namespaced_custom_object(
+                                    group=ce_config.DEFAULT_GROUP,
+                                    version=ce_config.DEFAULT_VERSION,
+                                    namespace=self.namespace,
+                                    plural="jobruns")
+        except ApiException as e:
+            logger.warning("Listing all jobruns failed with {} {}"
+                           .format(e.status, e.reason))
+            return
+
+        for jobrun in jobruns['items']:
+            try:
+                jobrun_name = jobrun['metadata']['name']
```

Comment:
@JosepSampe how we now it's completed job and not one in progress?

Reply:
@gilv ready